README.md

```diff
@@ -10,976 +10,22 @@ allows using docker containers for functional/integration/~~unit~~ testing.
 
 > TestContainers is a Java 8 library that supports JUnit tests, providing lightweight, throwaway instances of common databases, Selenium web browsers, or anything else that can run in a Docker container.
 
-[testcontainers-scala in action](http://dimafeng.com/2016/08/01/testcontainers-selenium)
+The following test frameworks are supported: 
```

Comment:
Curious if it'd make sense to move this to the usage doc or to the bottom of this doc

Reply:
My motivation for including it in the front page was that it's likely the first question a newcomer would ask: is this relevant for me, does it support my test framework? 
I'll move it down the doc and you'll let me know what you think